NaviGator/hardware_drivers/navigator_kill_board/nodes/kill_board_driver.py

```diff
@@ -267,12 +278,30 @@ def wrench_cb(self, msg):
                 )
             self.wrench = wrench
 
-    def network_cb(self, msg):
+    def network_kill_alarm_cb(self, alarm):
         """
         Pings kill board on every network hearbeat message. Pretends to be the rf-based hearbeat because
         real one does not work :(
         """
-        self.request(constants["PING"]["REQUEST"], constants["PING"]["RESPONSE"])
+        if alarm.raised:
+            self.nw_killed = True
+            if self.sw_killed:
```

Comment:
Unless I'm missing something, we could maybe expand `nw_killed` and `sw_killed` to their full name. I'm having a hard time understanding what the "nw" and "sw" mean.